apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Tests/LruCacheTest.cs

```diff
@@ -128,7 +128,7 @@ public void MultithreadedUse()
             }
             for (int i = 0; i < threadCount; i++)
             {
-                Assert.True(threads[i].Join(TimeSpan.FromSeconds(5)));
+                Assert.True(threads[i].Join(TimeSpan.FromSeconds(5).Milliseconds));
```

Comment:
Please remove `.Milliseconds`. Using `TimeSpan`s directly is preferable where possible.